diff -urN a/config.layout b/config.layout
--- a/config.layout	2018-02-09 18:30:35.000000000 +0800
+++ b/config.layout	2019-09-21 09:36:47.000000000 +0800
@@ -154,6 +154,30 @@
     proxycachedir: ${localstatedir}/cache/httpd/proxy
 </Layout>     
 
+# Layout used in openEuler httpd packaging.
+<Layout openEuler>
+    prefix:        /usr
+    exec_prefix:   ${prefix}
+    bindir:        ${prefix}/bin
+    sbindir:       ${prefix}/sbin
+    libdir:        ${prefix}/lib
+    libexecdir:    ${prefix}/libexec
+    mandir:        ${prefix}/man
+    sysconfdir:    /etc/httpd/conf
+    datadir:       ${prefix}/share/httpd
+    installbuilddir: ${libdir}/httpd/build
+    errordir:      ${datadir}/error
+    iconsdir:      ${datadir}/icons
+    htdocsdir:     /var/www/html
+    manualdir:     ${datadir}/manual
+    cgidir:        /var/www/cgi-bin
+    includedir:    ${prefix}/include/httpd
+    localstatedir: /var
+    runtimedir:    /run/httpd
+    logfiledir:    ${localstatedir}/log/httpd
+    proxycachedir: ${localstatedir}/cache/httpd/proxy
+</Layout>     
+
 #   According to the /opt filesystem conventions
 <Layout opt>
     prefix:        /opt/apache
